packages/ui/src/components/molecules/Gift/styles.scss

```diff
@@ -88,7 +88,7 @@
     }
   }
 
-  [data-fs-gift-icon] {
+  [data-fs-icon] {
```

Comment:
can't we keep this data-attr on the code? 🤔 

If not, it's okay, can u also remove it from the [doc](https://evergreen.faststore.dev/components/molecules/gift#customization) pls?

Reply:
yes we can, but we need to apply this styles to the `[data-fs-icon]`, so I prefer to use just `[data-fs-icon]` instead of `[data-fs-gift-icon] [data-fs-icon]` for specificity sake. Also, I kept this data-attr in doc because it's possible to use it yet. What do you think?

Reply:
hmm I think it works. But I think we should remove it from the doc cause we are not declaring it anywhere else.

Reply:
we declare [it here](https://github.com/vtex/faststore/blob/d0daa7b56b43bd0adbbb26f19a6f5885a7f1a1d0/packages/components/src/molecules/Gift/Gift.tsx#L22).

Reply:
sorry! I've missed that. Let's keep it. LGTM!

Reply:
NP hahaha :)